src/storage/mutate/AddVerticesProcessor.cpp

```diff
@@ -160,25 +179,23 @@ void AddVerticesProcessor::doProcessWithIndex(const cpp2::AddVerticesRequest& re
                 }
 
                 auto key = NebulaKeyUtils::vertexKey(spaceVidLen_, partId, vid, tagId);
+                if (ifNotExists_ && !visited.emplace(key).second) {
```

Comment:
What is the expected order when graph insert same (vId + tagId) with different value?

Reply:
in order of input, if encounter the same(vid + tagid) the do nothing when if_not_exists

Reply:
So when `ifNotExists_` is true, first one will be inserted, when `ifNotExists_` is false, the last one will be insert. Am I right?

Reply:
right